src/ValidationCheck.js

```diff
@@ -0,0 +1,53 @@
+class ValidationCheck {
+    static consistsOfPositiveNumber(str) {
+        const NUMBER_REGEXP = /^[1-9]+$/;
+        if (!NUMBER_REGEXP.test(str)) {
+          return false;
+        }
+        return true;
+    }
```

Comment:
이렇게 적어주면 boolean 값을 바로 리턴해줍니다!
ValidationCheck 클래스의 메서드들을 이런 식으로 바꿔주면 훨씬 가독성이 좋을 것 같다고 생각해요✨
```suggestion
    static consistsOfPositiveNumber(str) {
        const NUMBER_REGEXP = /^[1-9]+$/;
        return NUMBER_REGEXP.test(str);
    }
```

Reply:
와 정말 깔끔하네요 구현할 때는 왜 몰랐을까요..! 감사합니다!!

Reply:
좋은 코멘트인것 같아요 저도 동의합니다 👍